File: NetBeansProjects/WebApplication1/src/java/com/epic/canvascontrollibrary/CCLBarsMixedWithLabeledLineGraphProps.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.epic.canvascontrollibrary;

import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author Gilgamesh
 */
    public class CCLBarsMixedWithLabeledLineGraphProps
    {
        public String CanvasID;
        public String WindowID;
        public String X;
        public String Y;
        public String Width;
        public String Height;
        public List<Object> Data;
        public String MaxValue;
        public String NumMarksY;
        public String Title;
        public String TitleTextColor;
        public String TitleTextHeight;
        public String TitleTextFontString;
        public String BarWidth;
        public List<Object> BarLabelsWithBoundingBoxes;
        public String H;
        public String AxisLabelsTextHeight;
        public String AxisLabelsTextFontString;
        public String AxisLabelsTextColor;
        public String MarginLeft;
        public String GapBetweenBars;
        public String BarClickFunction;
        public String AlreadyUnregisteredAnimation;
        public String HasLegend;
        public String MarginRight;
        public List<Object> LinesData;
        public List<Object> LineXYs;
        public String LineClickFunction;
        public String YMaxValue;
        public Object Tag;
        public String TabStopIndex;

        CCLBarsMixedWithLabeledLineGraphProps()
        {
            Data = new ArrayList<Object>();
            BarLabelsWithBoundingBoxes = new ArrayList<Object>();
            LinesData = new ArrayList<Object>();
            LineXYs = new ArrayList<Object>();
        }
    }

File: ASP.NET MVC Web Api New Solution/CanvasControlLibrary/CanvasControlLibrary/Models/CCLBarsMixedWithLabeledLineGraphProps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;

namespace CanvasControlLibrary.Models
{
    public class CCLBarsMixedWithLabeledLineGraphProps
    {
        public class BarData
        {
            public string Label { get; set; }
            public float Value { get; set; }
            public string Color { get; set; }
        }

        public class BoundingBox
        {
            public int X { get; set; }
            public int Y { get; set; }
            public int Width { get; set; }
            public int Height { get; set; }
        }

        public class LineXY
        {
            public int X { get; set; }
            public int Y { get; set; }
        }

        public class LabelValue
        {
            public string Label;
            public float Value;
        }

        public class LineDataColor
        {
            public List<LabelValue> Values;
            public string LineColor;
        }

        public string CanvasID { get; set; }
        public int WindowID { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public List<BarData> Data { get; set; }
        public int MaxValue { get; set; }
        public int NumMarksY { get; set; }
        public string Title { get; set; }
        public string TitleTextColor { get; set; }
        public int TitleTextHeight { get; set; }
        public string TitleTextFontString { get; set; }
        public int BarWidth { get; set; }
        public List<BoundingBox> BarLabelsWithBoundingBoxes { get; set; }
        public int H { get; set; }
        public int AxisLabelsTextHeight { get; set; }
        public string AxisLabelsTextFontString { get; set; }
        public string AxisLabelsTextColor { get; set; }
        public int MarginLeft { get; set; }
        public int GapBetweenBars { get; set; }
        public int AlreadyUnregisteredAnimation { get; set; }
        public int HasLegend { get; set; }
        public int MarginRight { get; set; }
        public List<LineDataColor> LinesData { get; set; }
        public List<LineXY> LineXYs { get; set; }
        public int YMaxValue { get; set; }
        public object Tag { get; set; }
        public int TabStopIndex { get; set; }

        public CCLBarsMixedWithLabeledLineGraphProps()
        {
            Data = new List<BarData>();
            BarLabelsWithBoundingBoxes = new List<BoundingBox>();
            LinesData = new List<LineDataColor>();
            LineXYs = new List<LineXY>();
        }
    }
}